src/Systmm.f90

```diff
@@ -182,6 +183,7 @@ SUBROUTINE SYSTMM(temp_file)
 do nyear=start_year,end_year
   write(*,*) ' Simulation Year - ',nyear,start_year,end_year
   nd_year=365
+  [NEED A MORE GENERAL LEAP YEAR ROUTINE. I HAVE ONE IF NEEDED - JRY]
```

Comment:
John - this is your code. So if you have a better leap year routine, please add and do a pull request!